pkgs/top-level/all-packages.nix

```diff
@@ -20309,6 +20309,8 @@ with pkgs;
 
   xcpretty = callPackage ../development/tools/xcpretty { };
 
+  xcbeautify = callPackage ../by-name/xc/xcbeautify/package.nix { stdenv = gcc13Stdenv; };
+
```

Comment:
```suggestion
```

You don't need to add packages defined in `/by-name/` to `all-packages.nix`. 

Since you are only unpacking a zip file I assume you don't need to specify `stdenv = gcc13Stdenv;`, right?

Reply:
This should be fixed by moving the package instead.

Reply:
> This should be fixed by moving the package instead.

Why? as @siddarthkay said, they only added `stdenv = gcc13Stdenv;` to work around being detected as an unnecessary declaration, not because `gcc13Stdenv` is necessary.

Reply:
Thanks for clarifying!